cpp/src/ice2slice/Main.cpp

```diff
@@ -88,24 +88,24 @@ compile(const vector<string>& argv)
         return EXIT_SUCCESS;
     }
 
-    vector<string> cppArgs;
+    vector<string> preprocessorArgs;
```

Comment:
I renamed `cppArgs`, `cppHandle` and `icecpp` in all compilers for "better" names. That is `preprocessorArgs`, `preprocessedHandle` and `preprocessor` respectively.